pages/testproj/test_settings.py

```diff
@@ -186,6 +186,9 @@ def language_mapping(lang):
 COVERAGE_BRANCH_COVERAGE = False
 PAGE_ENABLE_TESTS = True
 
+PASSWORD_HASHERS = [
+    'django.contrib.auth.hashers.SHA1PasswordHasher'
```

Comment:
We could change it. I like changes to speedup development.
This is of course optional.

I was not involve in project for long time so will let @batiste to review it.